README.md

```diff
@@ -42,3 +42,57 @@ Open the Microsoft ODBC Administrator (64-bit) and verify that "MongoDB Atlas SQ
 
 ### 32-bit
 Follow the validation steps listed in the 64-bit section, but make sure to use the 32-bit Microsoft ODBC Administrator. Additionally, the registry keys should be listed under `HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\ODBC` instead of `HKEY_LOCAL_MACHINE\SOFTWARE\ODBC`, and the dll path must point to the 32-bit version of the driver.
+
+\<document field access expression\> ::= [\<compound identifier\>](#identifiers) \| [\<expression\>](#expressions) \".\" [\<compound identifier\>](#identifiers)</br>
+
+## Unsupported Functions
+
+The following ODBC functions are currently not supported by the driver. It is possible that some may be supported in future releases. Note that some unsupported ANSI functions have supported Unicode versions.
```

Comment:
```suggestion
The following ODBC functions are currently not supported by the driver. It is possible that some may be supported in future releases. Note that some unsupported ANSI functions have supported Unicode versions. If the version of the function suffixed with 'W', e.g., `SQLTables`, the Unicode version is supported.
```

Reply:
I thought about adding something similar, and agree it might be worthwhile!